logging/dw_es_domain.yaml

```diff
@@ -165,6 +171,16 @@ Resources:
         UpdatePolicy:
             EnableVersionUpgrade: true
 
+    # Keep endpoint in parameter store for easy lookup.
+
+    SsmParameter:
+        Type: AWS::SSM::Parameter
+        Properties:
+            Description: "Value of 'host:port' of Elasticsearch Service cluster for log processing"
+            Name: !Sub "/DW-ETL/ES-By-Env-Type/${EnvName}"
+            Type: "String"
+            Value: !Sub "${ElasticsearchDomain.DomainEndpoint}:443"
```

Comment:
looks good. presumably the dev and prod users have access to this SSM config.

Reply:
Indeed they do. The role used to run the log parsing has these permissions:
```
                          - Effect: "Allow"
                            Action: "ssm:GetParameter*"
                            Resource: "arn:aws:ssm:*:*:parameter/DW-ETL/*"
```